lib/linguist/languages.yml

```diff
@@ -5886,6 +5886,15 @@ RenderScript:
   tm_scope: none
   ace_mode: text
   language_id: 323
+Rez:
+  type: programming
+  extensions:
+  - ".r"
+  - ".rez"
```

Comment:
We need samples and search links for all extensions being added. Keep in mind our documented usage requirements so it might be better to hold off on adding a rarely used extension on the first attempt as an override can be used until usage meets requirements.

Reply:
ok so I'll just remove the `.rez` one, then

Reply:
removed in [7445872](https://github.com/github-linguist/linguist/pull/6524/commits/744587214c6ebda7f4eada0f9c8c1c27db0ac487).